### PR TITLE
updated zord and megaformZord headers.

### PR DESCRIPTION
**Pull request by guswadsworth12**
MFZ and Zord recieved new headers in this update. 

Review code

Test
Look at the new headers for MFZ and Zord

### CODE REVIEW

**Review by phildominguez — CHANGES_REQUESTED** [review]
Looks nice! One suggestion.

File: templates/actor/actor-zord-sheet.hbs [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
-  <header class="sheet-header">
+  <div class="sheet-header">
+    <span class="resource flex-group-center npc-vert-center">
     {{> "systems/essence20/templates/actor/parts/actor-base-header.hbs"}}
-    <div class="zord-header">
-      <input type="text" name="system.prerequisite" value="{{system.prerequisite}}"
-        placeholder="{{ localize 'E20.ZordPrerequisite' }}" />
-      <input type="text" name="system.ranger" value="{{system.ranger}}" placeholder="{{ localize 'E20.ZordRanger' }}" />
-      <input type="checkbox" name="system.isCombiner" {{checked system.isCombiner}} />
+    </span>
```
[the remark this answers is not in the document]
Comment (guswadsworth12) [reply templates/actor/actor-zord-sheet.hbs:6]: alright i'll make that change now

[the remark this answers is not in the document]
Comment (guswadsworth12) [reply templates/actor/actor-zord-sheet.hbs]: Fixed Please review

File: templates/actor/actor-megaformZord-sheet.hbs [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
-        {{> "systems/essence20/templates/actor/parts/actor-base-header.hbs"}}
-      </span>
-      <div class="pr-header-fields">
+  {{!-- Actor Sheet Header --}}
+  <div class="sheet-header">
+    <span class="resource flex-group-center npc-vert-center">
+      <div class="npc-image npc-vert-center">
+        <img class="profile-img" src="{{actor.img}}" data-edit="img" />
+      </div>
+    </span>
```
Comment (phildominguez) [inline templates/actor/actor-megaformZord-sheet.hbs:8]:
```suggestion
<header class="sheet-header">
  <img class="profile-img" src="{{actor.img}}" data-edit="img" title="{{actor.name}}" />
```
Clarifying code change for this and other sheets

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: templates/actor/actor-npc-sheet.hbs [hunk 3]
```diff
@@ -1,13 +1,15 @@
 <form class="{{cssClass}} {{actor.type}}" autocomplete="off">
   {{!-- Actor Sheet Header --}}
   <div class="sheet-header">
-    <span class="resource flex-group-center">
-      {{> "systems/essence20/templates/actor/parts/actor-base-header.hbs"}}
+    <span class="resource flex-group-center npc-vert-center">
+      <div class="npc-image npc-vert-center">
+        <img class="profile-img" src="{{actor.img}}" data-edit="img" />
+      </div>
```
Comment (phildominguez) [inline templates/actor/actor-npc-sheet.hbs:7]: Don't forget this guy! Should be similar to the other headers.

File: sass/actors/_npc.scss [hunk 4]
```diff
@@ -1,9 +1,10 @@
 @use '../assets/variables' as v;
 
 .essence20 .npc-sheet-header {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+  color: v.$textcolor1;
```
Comment (phildominguez) [inline sass/actors/_npc.scss:7]: This can be removed since NPC will be like the others

File: sass/actors/_npc.scss [hunk 5]
```diff
@@ -29,3 +30,10 @@
   border-radius: v.$border-radius;
   box-shadow: v.$boxshadow;
 }
+
+.essence20 .npc-vert-center {
+  display: flex;
+  align-items: center;
+  margin-top: auto;
+  margin-bottom: auto;
+}
```
Comment (phildominguez) [inline sass/actors/_npc.scss:39]: And this

**Review by phildominguez — CHANGES_REQUESTED** [review]

**Review by phildominguez — APPROVED** [review]
🚢 🚢 🚢

**Pull request merged**